src/main/webapp/app/account/register/register.component.html

```diff
@@ -1,8 +1,15 @@
-<div>
+<div *ngIf="isRegistrationEnabled">
     <div class="row justify-content-center">
         <div class="col-md-8">
             <h1 jhiTranslate="register.title">Registration</h1>
 
+            <!-- TEST CODE -->
+            <!--            <div class="form-group">-->
+            <!--                <label style="color: red" class="form-control-label">Email Pattern for Testing</label>-->
+            <!--                <input style="color: red" class="form-control" id="regex" name="regex" [(ngModel)]="allowedEmailPattern" (ngModelChange)="updateAllowedEmailPattern()" />-->
+            <!--            </div>-->
+            <!-- TEST CODE -->
+
```

Comment:
Is this still needed?
```suggestion
```

Reply:
It really helped me in testing the regex pattern, so this might be super useful for other administrators playing around with Artemis, therefore I would like to keep it.